addons/calendar/static/src/js/calendar_view.js

```diff
@@ -72,6 +72,11 @@ var AttendeeCalendarRenderer = CalendarRenderer.extend({
         }
     },
 
+    init: function (){
+        this._super.apply(this, arguments);
+        // partener_id filter is render even if invisible="1"
+        delete this.state.filters.partner_id
```

Comment:
Not sure why you need this, looks like a dirty trick.
Can you explain a bit more?

Reply:
I find a better way by removing filters from xml